core/components/organisms/textField/__stories__/withTextArea.story.jsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { TextField } from '@/index';
+import { TextFieldWithTextarea } from '../TextFieldWithTextarea';
+
+// CSF format story
+export const withTextArea = () => {
+  return (
+    <div className="w-25">
+      <TextField withTextarea={true} />
+    </div>
+  );
+};
+
+const customCode = `() => {
+  const [inputText, setInputText] = React.useState('');
+  const [errorState, setErrorState] = React.useState(false);
+  const [helpText, setHelpText] = React.useState('');
+  const max = 50;
+  React.useEffect(() => {
+    if (inputText.length > max) {
+      setErrorState(true);
+      setHelpText('I am in error state');
+    } else {
+      setErrorState(false);
+      setHelpText('I am the helptext');
+    }
+  }, [inputText]);
+  return (
+    <div className="w-25">
+      <TextField
```

Comment:
Please add a newline and update the helptext message correctly depending upon the character limit